### ISSUE TITLE
[Feature request] Ability to merge different datasets

### ISSUE THREAD

**Issue by JoQCcoz** [nextpyp/next#4]
It would be nice to have the ability to merge particles or imports during the projects workflow. 

For example, we have 2 datasets of lamellaes from the same types samples collected at different times. We cannot merge the data before import since they have different gain references.

**Comment by abartesaghi** [comment]
For merging datasets, there is a related issue on the [main discussions page](https://github.com/orgs/nextpyp/discussions/88).
